app/src/main/java/com/dara/fitnessapp/ui/composables/CustomText.kt

```diff
@@ -22,14 +23,17 @@ fun CustomText(
     paddingTop: Dp = 0.dp,
     paddingEnd: Dp = 0.dp,
     textColor: Color = Color.Black,
-    textAlign: TextAlign? = null
+    textAlign: TextAlign = TextAlign.Start,
+    textDecoration: TextDecoration =  TextDecoration.None,
+    action: () -> Unit = {}
 ) {
     Text(
```

Comment:
Why do we have this composable aside from Compose's Text?
Are we gonna have more CustomTexts?